render/hyperscript.js

```diff
@@ -108,6 +104,7 @@ function hyperscript(selector) {
 	var normalized = Vnode.normalizeChildren(children)
 
 	if (typeof selector === "string") {
+		var cached = selectorCache[selector] || compileSelector(selector)
```

Comment:
Mind combining this with the next line, to save a temporary variable? (Your patch is more so optimizing space than speed, BTW.)